integration_tests/test_env_populator.py

```diff
@@ -0,0 +1,31 @@
+from robyn.env_populator import load_vars, parser
+import pathlib
+import os
+
+# content of test_tmp_path.py
+CONTENT = "PORT=8080"
+
+path = pathlib.Path(__file__).parent  
+
+#create a directory before test and delete it after test
+def test_create_file():
```

Comment:
You can use fixtures(https://docs.pytest.org/en/6.2.x/fixture.html) instead of depending on the execution order of tests.

You would write the test like this

```
@pytest.fixture
def env_file():
    dir = path / "test_dir"
    env_file = dir / "robyn.env"
    env_file.write_text(CONTENT)
    yield
    env_file.unlink()

def test_parser(env_file):
     ...
```